scripts/variantstore/docs/aou/vds/cluster/AoU Delta VDS Cluster Configuration.md

```diff
@@ -1,6 +1,6 @@
-# Delta VDS Hail Cluster Configuration
+# VDS Hail Cluster Configuration
```

Comment:
The doc still has 'Delta' in its name despite the removal of 'Delta' here.

Though it might make it more clear to the reader that these instructions were what we used at the time of Delta are no longer part of our production pipeline.

Reply:
We are keeping this document around in case we need to do anything with the VDS output, which is not tied to one callset.